src/test/java/com/google/neighborgood/IntegrationTest.java

```diff
@@ -256,46 +260,24 @@ public void _06_Userpage_AsLoggedHelper_CompleteTask() {
     // Location of first task listed in the complete task table in userpage
     String taskCompletedXPath = "//tbody[@id='complete-task-body']/tr[1]";
 
-    String taskOverview =
-        wait.until(
-                new Function<WebDriver, WebElement>() {
-                  public WebElement apply(WebDriver driver) {
-                    return driver.findElement(By.xpath(taskCompletedXPath + "/td[1]"));
-                  }
-                })
-            .getText();
-    assertEquals(recentTask.get("overview"), taskOverview);
-
-    String taskStatus =
-        wait.until(
-                new Function<WebDriver, WebElement>() {
-                  public WebElement apply(WebDriver driver) {
-                    return driver.findElement(By.xpath(taskCompletedXPath + "/td[2]"));
-                  }
-                })
-            .getText();
-    assertEquals(recentTask.get("status"), taskStatus);
-
-    String taskNeighbor =
-        wait.until(
-                new Function<WebDriver, WebElement>() {
-                  public WebElement apply(WebDriver driver) {
-                    return driver.findElement(By.xpath(taskCompletedXPath + "/td[3]"));
-                  }
-                })
-            .getText();
-    assertEquals(recentTask.get("nickname"), taskNeighbor);
-
-    String taskPoints =
-        wait.until(
-                new Function<WebDriver, WebElement>() {
-                  public WebElement apply(WebDriver driver) {
-                    return driver.findElement(By.xpath(taskCompletedXPath + "/td[4]"));
-                  }
-                })
-            .getText();
-    assertEquals(recentTask.get("points"), taskPoints);
-
+    // Content targets that will be verified
+    String[] taskContentsTarget = {"overview", "status", "nickname", "points"};
+
+    // Iterates over task elements and compares the element's text with that of the stored
+    // recentTask
+    for (int i = 0; i < taskContentsTarget.length; i++) {
+      String taskContentItem =
+          wait.until(
+                  new Function<WebDriver, WebElement>() {
+                    public WebElement apply(WebDriver driver) {
+                      return driver.findElement(
+                          By.xpath(taskCompletedXPath + "/td[" + (i + 1) + "]"));
```

Comment:
Personally I don't prefer adding an int directly with String, although it works : )

Reply:
Also, since we add 1 to i during each loop, why not start the loop from i = 1?

Reply:
> Also, since we add 1 to i during each loop, why not start the loop from i = 1?

I thought about that too, but then in the assertEquals statement the array would have an index of [i-1] and the condition in the for loop would also have to have a 1 added, so either way I'd have to add or subtract 1. Personally I prefer this approach.

Reply:
> Personally I don't prefer adding an int directly with String, although it works : )

I prefer leaving it without explicitly converting it to a string since string concatenation in Java takes care of that for me :)
But I'll keep this is mind for when using other languages in the future!